src/utils/helpers/application/getSectionsDetails.ts

```diff
@@ -26,6 +26,7 @@ const getSectionDetailsArray = (sectionsList: TemplateSection[]): SectionDetails
     const totalPages = pageBreaks.length + 1
 
     return {
+      active: true,
```

Comment:
I assume this is set to `true` initially, because we haven't yet been able to evaluate the visibility conditions of the elements?

Reply:
Sure. I can change that

Reply:
I didn't mean to change it, was just a question.